docs/content/preview/drivers-orms/go/pg.md

```diff
@@ -66,68 +75,288 @@ db := pg.Connect(opt)
 
 | Parameter | Description | Default |
 | :---------- | :---------- | :------ |
-| user | user for connecting to the database | yugabyte
-| password | password for connecting to the database | yugabyte
-| host  | hostname of the YugabyteDB instance | localhost
-| port |  Listen port for YSQL | 5433
-| dbname | database name | yugabyte
-| sslMode | SSL mode | require
+| user | User for connecting to the database | yugabyte |
+| password | User password | yugabyte |
+| host  | Hostname of the YugabyteDB instance | localhost |
+| port |  Listen port for YSQL | 5433 |
+| dbname | Database name | yugabyte |
+| sslMode | SSL mode | require |
+
+#### Use SSL
+
+You can choose to enable or disable SSL for your local YugabyteDB cluster. To configure SSL/TLS for your YugabyteDB cluster, refer to [Enable client-to-server encryption](../../../secure/tls-encryption/client-to-server/). YugabyteDB Managed requires SSL connections, and SSL/TLS is enabled by default for client-side authentication.
+
+#### CA certificate
+
+Use the [CA certificate](../../../secure/tls-encryption/server-certificates/#generate-the-root-certificate-file) generated above as part of the SSL/TLS configuration of your cluster.
+
+In case of a YugabyteDB Managed cluster, to download the CA certificate for your cluster in YugabyteDB Managed, do the following:
+
+1. On the **Clusters** tab, select a cluster.
+
+1. Click **Connect**.
+
+1. Click **Connect to your application** and download the CA cert.
+
+#### OpenSSL
+
+Install [OpenSSL](https://www.openssl.org/) 1.1.1 or later only if you have a YugabyteDB setup with SSL/TLS enabled. YugabyteDB Managed clusters are always SSL/TLS enabled.
 
-### Step 3: Create tables
+The following table summarizes the SSL modes and their support in the driver:
```

Comment:
Move this stuff to under Use SSL; delete the table, just say the driver supports all SSL modes; and then the YBM bit